### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by ghost** [phalcon/phalcon#74] [Q&A] [answered]
Will CHANGELOG.md be maintained for v5? This is very useful for those of us who follow development progress.

**Comment by niden** [comment, marked as answer]
Yes the changelog will always be maintained to ensure that not only the community, but the team itself knows about features and when they were introduced.

At the moment we are pretty much _copying_ code from Zephir to PHP with no additional changes - if any. Once v5 is out, the changelog will become more relevant
